apps/portal-api/src/modules/services/definition/service-capability/service-capability.domain.test.ts

```diff
@@ -0,0 +1,145 @@
+import { v4 as uuidv4 } from 'uuid';
+import { afterEach, beforeEach, describe, expect, it } from 'vitest';
+import { dbUnsecure } from '../../../../../knexfile';
+import { contextAdminUser } from '../../../../../tests/tests.const';
+import ServiceCapability, {
+  ServiceCapabilityId,
+} from '../../../../model/kanel/public/ServiceCapability';
+import ServiceDefinition, {
+  ServiceDefinitionId,
+} from '../../../../model/kanel/public/ServiceDefinition';
+import { loadServiceCapabilitiesBy } from './service-capability.domain';
+
+describe('Service Capability domain', () => {
+  let testServiceDefinitionId: ServiceDefinitionId;
+  let testServiceDefinitionId2: ServiceDefinitionId;
+  let testCapabilityIds: ServiceCapabilityId[] = [];
+  let testServiceDefIds: ServiceDefinitionId[] = [];
+
+  beforeEach(async () => {
+    const serviceDefinition = await dbUnsecure<ServiceDefinition>(
+      'ServiceDefinition'
+    )
+      .insert([
+        {
+          id: uuidv4() as ServiceDefinitionId,
+          name: 'Test Service Definition',
+          description: 'Test service definition for capability tests',
+        },
+        {
+          id: uuidv4() as ServiceDefinitionId,
+          name: '2 Test Service Definition 2',
+          description: '2 Test service definition for capability tests2 ',
+        },
+      ])
+      .returning('*');
+
+    testServiceDefinitionId =
+      serviceDefinition[0]?.id ?? (uuidv4() as ServiceDefinitionId);
+    testServiceDefinitionId2 =
+      serviceDefinition[1]?.id ?? (uuidv4() as ServiceDefinitionId);
+
+    const capabilities = await dbUnsecure<ServiceCapability>(
+      'Service_Capability'
+    )
+      .insert([
+        {
+          id: uuidv4() as ServiceCapabilityId,
+          name: 'Test Capability 1',
+          description: 'First test capability',
+          service_definition_id: testServiceDefinitionId,
+        },
+        {
+          id: uuidv4() as ServiceCapabilityId,
+          name: 'Test Capability 2',
+          description: 'Second test capability',
+          service_definition_id: testServiceDefinitionId,
+        },
+        {
+          id: uuidv4() as ServiceCapabilityId,
+          name: 'Test Capability 3',
+          description: 'Third test capability',
+          service_definition_id: testServiceDefinitionId2,
+        },
+      ])
+      .returning('*');
+
+    testCapabilityIds = capabilities.map((cap) => cap.id);
+    testServiceDefIds = serviceDefinition.map((def) => def.id);
+  });
+
+  afterEach(async () => {
+    if (testCapabilityIds.length > 0) {
+      await dbUnsecure<ServiceCapability>('Service_Capability')
+        .whereIn('id', testCapabilityIds)
+        .delete();
+    }
+
+    if (testServiceDefinitionId) {
+      await dbUnsecure<ServiceDefinition>('ServiceDefinition')
+        .whereIn('id', testServiceDefIds)
+        .delete();
+    }
+  });
```

Comment:
I think this is not necessary and we could use a `beforeAll` instead of a `beforeEach`